src/components/list/OrderedList.stories.mdx

```diff
@@ -21,25 +21,27 @@ import ListItem from './ListItem.tsx';
   argTypes={{
     size: {
       control: 'select',
+      options: ['sm', 'md'],
```

Comment:
Do you need to explicitly provide options? I think they are picked-up automatically by storybook

Reply:
yes. It was not working before: https://main-components-pkg.branch.autoscout24.dev/?path=/story/components-data-display-list-unordered-list--unordered-list

Reply:
![Bildschirm­foto 2023-04-06 um 09 25 50](https://user-images.githubusercontent.com/71456764/230304952-b2bce682-7f4a-4564-a42a-01c77ba66d19.png)

Reply:
That's strange. A difference between compiled and dev run of storybook maybe. Good catch 👍🏻